src/util/mapbox.js

```diff
@@ -17,6 +17,10 @@ function makeAPIURL(urlObject: UrlObject, accessToken: string | null | void): st
     urlObject.protocol = apiUrlObject.protocol;
     urlObject.authority = apiUrlObject.authority;
 
+    if (apiUrlObject.path !== '/') {
+        urlObject.authority = `${urlObject.authority}${apiUrlObject.path}`;
```

Comment:
@jfirebaugh Changes made - we now prepend to the `urlObject.path` instead of appending to `urlObject.authority`.